common/src/mbus_api/mbus_nats.rs

```diff
@@ -37,6 +37,11 @@ pub fn bus() -> DynBus {
     )
 }
 
+/// Get the global timeout_opts
+pub fn timeout_opts() -> TimeoutOptions {
```

Comment:
Where does this function get used?

Reply:
We need this in the tests, but yeah I am working on removing it.